README.md

```diff
@@ -39,58 +39,58 @@ Multicall3 contains the following improvements over prior multicall contracts:
+| Fantom Testnet          | 4002                               | [0xcA11bde05977b3631167028862bE2a173976CA11](https://testnet.ftmscan.com/address/0xcA11bde05977b3631167028862bE2a173976CA11#code)                                |
+| Fantom Opera            | 250                                | [0xcA11bde05977b3631167028862bE2a173976CA11](https://ftmscan.com/address/0xcA11bde05977b3631167028862bE2a173976CA11#code)                                        |
+| BNB Smart Chain         | 56                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://bscscan.com/address/0xcA11bde05977b3631167028862bE2a173976CA11#code)                                        |
+| BNB Smart Chain Testnet | 97                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://testnet.bscscan.com/address/0xcA11bde05977b3631167028862bE2a173976CA11#code)                                |
+| Moonbeam                | 1284                               | [0xcA11bde05977b3631167028862bE2a173976CA11](https://moonscan.io/address/0xca11bde05977b3631167028862be2a173976ca11#code)                                        |
+| Moonriver               | 1285                               | [0xcA11bde05977b3631167028862bE2a173976CA11](https://moonriver.moonscan.io/address/0xca11bde05977b3631167028862be2a173976ca11#code)                              |
+| Moonbase Alpha Testnet  | 1287                               | [0xcA11bde05977b3631167028862bE2a173976CA11](https://moonbase.moonscan.io/address/0xca11bde05977b3631167028862be2a173976ca11#code)                               |
+| Harmony                 | 1666600000                         | [0xcA11bde05977b3631167028862bE2a173976CA11](https://explorer.harmony.one/address/0xca11bde05977b3631167028862be2a173976ca11?activeTab=7)                        |
+| Cronos                  | 25                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://cronoscan.com/address/0xca11bde05977b3631167028862be2a173976ca11#code)                                      |
+| Fuse                    | 122                                | [0xcA11bde05977b3631167028862bE2a173976CA11](https://explorer.fuse.io/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                              |
+| Songbird Canary Network | 19                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://songbird-explorer.flare.network/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)               |
+| Coston Testnet          | 16                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://coston-explorer.flare.network/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                 |
+| Coston2 Testnet         | -                                  | [0xcA11bde05977b3631167028862bE2a173976CA11](https://coston2-explorer.flare.network/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                |
```

Comment:
Using the RPC of https://coston2-api.flare.network/ext/C/rpc, looks like this is 114